.github/workflows/ios-custom-build.yml

```
@@ -30,6 +30,7 @@ on:
           - collab
           - synergy
           - mec
+          -ph4h
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)